implementations/rust/ockam/ockam_node/Cargo.toml

```diff
@@ -52,6 +52,7 @@ tokio = { version = "1.8", default-features = false, optional = true, features =
     "rt-multi-thread",
     "macros",
 ] }
+futures = "0.3.10"
```

Comment:
Hmm, I think we may need to disable some features for no_std?

Reply:
@thomcc interesting, CI should check that no_std build is working